sig-community-experience/charter.md

```diff
@@ -0,0 +1,94 @@
+# SIG YOURSIG Charter
+* prefilled
+This charter adheres to the conventions described in the [Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+## Scope
+
+The goal of this SIG is to reach and maintain a high quality of our services.
+Quality is so far defined as in the easy sentence "what helps the community and the user to solve a problem".
+Services needs to be understandable, easy to use, recreatable and doing a good job.
+High quality therefore can be defined by (not necessarly a complete or static list):
+easy to understand (know what you get before trying)
+easy to use (worth a try)
+easy to repeat (lowering the hurdle for using it again)
+solving a problem (knowing the problem)
+solving it good (knowing the community/customer)
+solving it fast (optimized, automated, workflow)
```

Comment:
Formatting: add `-` to make these a list with bullet points:
```suggestion

- easy to understand (know what you get before trying)
- easy to use (worth a try)
- easy to repeat (lowering the hurdle for using it again)
- solving a problem (knowing the problem)
- solving it good (knowing the community/customer)
- solving it fast (optimized, automated, workflow)
```

Reply:
thanks